qa/tasks/cephfs/test_full.py

```diff
@@ -375,8 +375,8 @@ class TestQuotaFull(FullnessTestCase):
     """
     Test per-pool fullness, which indicates quota limits exceeded
     """
-    pool_capacity = 1024 * 1024 * 32   # arbitrary low-ish limit
-    fill_mb = pool_capacity / (1024 * 1024)
+    pool_capacity = 1024 * 1024 * 32  # type: ignore # arbitrary low-ish limit
```

Comment:
again, which error do you get here?

Reply:
`error: Incompatible types in assignment (expression has type "int", base class "FullnessTestCase" defined the type as "None")`

Reply:
proper fix would be do change FullnessTestCase:

```python
class FullnessTestCase:
    pool_capacity = None  # type: Optional[int]
```

Reply:
Done. Thanks!